meta-ros-common/conf/layer.conf

```diff
@@ -25,3 +25,5 @@ LAYERDEPENDS_ros-common-layer = " \
 LAYERSERIES_COMPAT_ros-common-layer = "${ROS_OE_RELEASE_SERIES}"
 
 require conf/ros-distro/ros-distro.conf
+
+ROS_BRANCH_DEPRECATED := "${@bb.warn('You are building from *-draft branch, please switch to branch without a -draft suffix, see https://github.com/ros/meta-ros/issues/670')}"
```

Comment:
You are building from *-draft branch, please switch to branch without a -draft suffix, see https://github.com/ros/meta-ros/issues/670' ->
You are building from a *-draft branch, which has been deprecated and will be dropped as part of Milestone 14 (see https://github.com/ros/meta-ros/wiki/Superflore-OE-Recipe-Generation-Scheme#milestone-14). Please switch to the corresponding branch without the -draft suffix.'